pkg/render/compliance.go

```diff
@@ -49,12 +49,11 @@ const (
 	ComplianceServerKeyName    = "tls.key"
 
 	complianceServerTLSHashAnnotation = "hash.operator.tigera.io/tls-certificate"
-	ManagerTLSHashAnnotation          = "hash.operator.tigera.io/manager-certificate"
```

Comment:
Looks like this isn't being used anymore, so I thought it would be better to remove it.